app/lib/update_patients_from_pds.rb

```diff
@@ -21,14 +23,26 @@ def call
           PatientNHSNumberLookupJob.set(
             priority:,
             queue:,
-            wait: index * wait_between_jobs
+            wait: jobs_queued * wait_between_jobs
```

Comment:
How large do you think `patients` could be? If the collection was large, later jobs could have very significant delays.

Reply:
I agree we need to think about this, but maybe we just need to deal with this until we can get Sidekiq in place, which I'd like to start on really soon.

Reply:
While I agree we have this problem, I don't think this change makes it significantly worse. Most parents _don't_ have pending changes, and therefore there won't be that many additional jobs queued.